test/components/results/candidate-results/personality-dimension-columns.test.js

```diff
@@ -0,0 +1,42 @@
+import {Component} from "components/results/candidate-results/personality-dimension-columns";
+import ComponentHandler from "support/component-handler";
+import assessment from "support/json/assessment/dimension-based.json";
+
+jest.mock("lib/with-traitify", () => ((value) => value));
+
+describe("PersonalityDimensionColumns", () => {
```

Comment:
Also looks like there are a few warnings with props when running the tests. With this and archetype